dev/doc/changes.txt

```diff
@@ -244,6 +244,10 @@ define_evar_* mostly used internally in the unification engine.
   ...
   let Sigma (xn, sigma, pn) = ... in
   Sigma (ans, sigma, p1 +> ... +> pn)
+  
+  Examples of `Sigma.Unsafe.of_evar_map` include:
+  
+  Evarutil.new_evar env (Tacmach.project goal) ty   ----> Evarutil.new_evar env (Sigma.Unsafe.of_evar_map (Tacmach.project goal)) ty
```

Comment:
If you want to port things easily, yes. Otherwise, we should rather indicate that unsafe primitives should be avoided at the cost of having to rewrite your code using the proper Sigma API.